src/test/skript/tests/syntaxes/effects/EffPush.sk

```diff
@@ -21,4 +21,8 @@ test "pushing and pulling":
 	push {_e} along vector(-2,-1,-1)
 	assert {_e}'s velocity is vector(0,0,0) with "pig's velocity after push was wrong"
 
+	set the velocity of player to vector(0,0,0)
+	pull {_e} along vector(1,2,3)
+	assert {_e}'s velocity is vector(1,2,3) with "pull syntax didn't work D:"
```

Comment:
```suggestion
	set the velocity of player to vector(0,0,0)
	pull {_e} along vector(1,2,3)
	assert {_e}'s velocity is vector(1,2,3) with "pig's velocity after pull was wrong"
```
be descriptive in the error messages. They're what you're going to rely on when things fail, you don't want it saying "oopsy woopsy something went wrong". You want it saying exactly what went wrong.